docs/docs/containers/CONFLUENCE.md

```diff
@@ -160,6 +160,16 @@ of the setup. They can be controlled via the below environment variables.
   in a clustered load balancer architecture where the load balancer address changes depending on
   which node proxies the connection. If not specified, no internal proxies will be trusted.
 
+
+* `ATL_TOMCAT_ADDITIONAL_CONNECTOR_PORT` - mandatory, no default. If unset, no additional connector is defined in server.xml
```

Comment:
Is it mandatory? We have the if statement not to include if it is empty.
https://bitbucket.org/atlassian-docker/docker-atlassian-confluence-server/pull-requests/188/diff#chg_config/server.xml.j2_newline24